appveyor.yml

```diff
@@ -0,0 +1,44 @@
+build: off
+
+before_test:
+# http://help.appveyor.com/discussions/problems/6312-curl-command-not-found
+- set PATH=C:\Program Files\Git\mingw64\bin;%PATH%
+
+- curl -sS -ostack.zip -L --insecure http://www.stackage.org/stack/windows-i386
+- 7z x stack.zip stack.exe
+
+clone_folder: "c:\\stack"
+environment:
+  matrix:
+  - win_runtime: win-x64
```

Comment:
We always download the i368 (32 bit) version of stack, which I think means it will always use the 32-bit version of ghc, which also means that it will be compiling 32-bit binaries even if running on a 64-bit system.  I think you would have to use the 64-bit version of stack to actually build 64-bit binaries.

For elm-format (and I believe for Elm itself), I only build 32-bit Windows binaries, and no one has complained about this yet (they work fine on 64-bit Windows).  So you might just want to remove the 64-bit build.